fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/ReindexService.java

```diff
@@ -118,17 +118,23 @@ public void indexOcflObject(final String txId, final String ocflId) {
                                     String.format("Resource %s must have a parent defined", fedoraId.getFullId()));
                         }
                     }
-                    if (!headers.getInteractionModel().equals(NON_RDF_SOURCE.toString())) {
-                        final Optional<InputStream> content = session.readContent(fedoraId.getFullId())
-                                .getContentStream();
-                        if (content.isPresent()) {
-                            final RdfStream rdf = parseRdf(fedoraId, content.get());
-                            this.referenceService.updateReferences(txId, fedoraId, null, rdf);
+                    final var created = headers.getCreatedDate();
+                    if (!headers.isDeleted()) {
+                        if (!headers.getInteractionModel().equals(NON_RDF_SOURCE.toString())) {
+                            final Optional<InputStream> content = session.readContent(fedoraId.getFullId())
+                                    .getContentStream();
+                            if (content.isPresent()) {
+                                final RdfStream rdf = parseRdf(fedoraId, content.get());
```

Comment:
Recognizing that this is not new code, I believe this input stream is not being closed.